src/components/Swap/Approval/index.tsx

```diff
@@ -10,7 +10,7 @@ export interface ApprovalProps {
 }
 
 const ApprovalSubmit = ({ token }: ApprovalProps): JSX.Element | null => {
-  const { router } = useGetTenantData(nablaConfig) || {};
+  const { router } = useGetTenantData(nablaConfig)[0] || {};
```

Comment:
Can we maybe make this more expressive?

Reply:
I see that it's convenient to use it this way, but it's hard to understand from the naming etc. why you have to access the first part of an array. Maybe we just remove the `tenantName` from the array and only return the data of the current tenant here? The `tenantName` probably is not required most of the time.

Reply:
Sure thing

Reply:
Updated